### PR TITLE
Feat/show more

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

File: src/js/controllers/index.js [hunk 1]
```diff
@@ -936,15 +939,14 @@ angular.module('copayApp.controllers').controller('indexController', function($r
...
-    self.historyShowShowAll = false;
-    self.historyRendering = true;
+  self.showMore = function() {
     $timeout(function() {
-      $rootScope.$apply();
-      $timeout(function() {
-        self.historyRendering = false;
-        self.txHistory = self.completeHistory;
-      }, 100);
+      self.txHistory = self.txHistory.concat(self.completeHistory.slice(self.skipTxHistory, self.nextTxHistory));
```
Comment (matiu) [inline src/js/controllers/index.js]: Please remove skipTxHistory and only use nextTxHistory.
and
   self.txHistory = self.completeHistory.slice(0, self.nextTxHistory);

File: src/js/controllers/index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ angular.module('copayApp.controllers').controller('indexController', function($r
   self.usePushNotifications = self.isCordova && !isMobile.Windows();
   self.onGoingProcess = {};
   self.historyShowLimit = 10;
+  self.moreTx = 100;
```
Comment (matiu) [inline src/js/controllers/index.js]: should be: historyShowMoreLimit

**Comment by JDonadio** [comment]
Updated.

**Comment by matiu** [comment]
LGTM

**Comment by JDonadio** [comment]
No longer works in master branch

**Pull request closed without merging**